packages/datadog-plugin-winston/test/index.spec.js

```diff
@@ -203,14 +204,42 @@ describe('Plugin', () => {
               const index = semver.intersects(version, '>=3') ? 0 : 2
               const record = log.firstCall.args[index]
 
-              expect(record).to.be.an('error')
+              assert(record instanceof Error)
               expect(error).to.not.have.property('dd')
               expect(spy).to.have.been.calledWithMatch(meta.dd)
             })
             expect(await logServer.logPromise).to.include(meta.dd)
           })
 
           if (semver.intersects(version, '>=3')) {
+            it('should support sets and getters', async () => {
```

Comment:
```suggestion
            it('should support sets with getters', async () => {
```

Reply:
Ah, ok. I misunderstood. I thought it was specifically getters on a set. Carry on 👍